Source/MSBuildTools.Unity/Packages/com.microsoft.msbuildforunity/Editor/ProjectGenerator/MSBuildTemplates/UnixNetSupport.props

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information.
+https://github.com/dotnet/fsharp/blob/master/fcs/netfx.props (license is MIT)
+ -->
+<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <!-- When compiling .NET SDK 2.0 projects targeting .NET 4.x on Mono using 'dotnet build' you -->
+    <!-- have to teach MSBuild where the Mono copy of the reference asssemblies is -->
+    <TargetIsMono Condition="($(TargetFramework.StartsWith('net4')) OR $(TargetFrameworkVersion.StartsWith('v4.'))) AND '$(OS)' == 'Unix'">true</TargetIsMono>
```

Comment:
>TargetIsMono [](start = 5, length = 12)

For the sake of protecting against breaking changes in the future I recommend naming new Properties/ItemGroups/Targets/etc. in the style of privates (starting with an underscore).

Reply:
So this file was taken almost verabitim from https://github.com/dotnet/fsharp/blob/master/fcs/netfx.props, and I don't wish to further change it apart from adding the link.